comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -170,6 +170,7 @@
       <!--
         Do not edit this section manually and use ./comps_doc.sh script
       -->
+      <packagereq type="default">rubygem-openscap_parser</packagereq>
```

Comment:
shouldn't this be before the comment? did we break `add_to_comps.rb`? @ekohl

Reply:
Quite likely :(